clipmenud

```diff
@@ -106,6 +108,12 @@ if ! (( has_clipnotify )); then
     echo "WARN: See https://github.com/cdown/clipnotify." >&2
 fi
 
+command -v xdotool >/dev/null 2>&1 && has_xdotool=1
+
+if [[ "$CM_IGNORE_WINDOW" && ! (( has_xdotool )) ]]; then
```

Comment:
You can't use `((` inside `[[` like that, it will always be true :-) You need to use `((`in keyword-eligible scope

Reply:
You want `if [[ $CM_IGNORE_WINDOW ]] && (( has_xdotool )); then` :-)

Reply:
Thanks!
```suggestion
if [[ $CM_IGNORE_WINDOW ]] && ! (( has_xdotool )); then
```